src/test/cypress/support/commands.ts

```diff
@@ -36,7 +36,7 @@ declare global {
             logout(): any;
             loginWithGUI(credentials: CypressCredentials): any;
             getSettled(selector: string, options?: {}): Chainable<Cypress>;
-            reloadUntilFound(selector: string): Chainable<Cypress>;
+            reloadUntilFound(selector: string): Chainable<undefined>;
```

Comment:
To be honest i dont understand this change. Why undefined? Why not any? Can you explain?

Reply:
Ah, yeah that was an unintended change from previous testing. I reverted it.